admin/release.py

```diff
@@ -202,6 +205,12 @@ def publish_docs(flocker_version, doc_version, environment):
                      prefix=version_prefix,
                      keys=existing_version_keys - new_version_keys))
 
+
+    # Update the key used for error pages
+    yield Effect(
```

Comment:
Seems like most likely not.  As I understand things, this `Effect` will cause the error page (404 or otherwise?) for the entire `docs.clusterhq.com` site to be changed to the 404 error page in the package being released by this code.

If this code were only updating the error page for the version of the documentation corresponding to the release being made, it would make sense to do the update (I don't know off hand whether that's actually possible - I could imagine being allowed to specify an error page to handle errors that occur below some prefix, but I haven't looked at CloudFront to learn if that's actually supported or not).  I don't think it makes sense to do the update for all error pages on the entire site - since the point of dev releases isn't that we have something polished that we want users to see, just that we want to exercise the release process.

Of course, if we don't do this update then we're not exercising this part of the release process...  So that's a bit sad.  We do having the staging bucket, though.  So probably that's how we'll exercise this code?

Long story short, I agree that it makes sense to skip this update if `is_dev` is true.